README.md

```diff
@@ -0,0 +1,16 @@
+# radicle-upstream
+
+A monorepo for related Radicle projects:
+
+- `app`: Cross-platform desktop app, currently used for product explorations.
+- `proxy`: Intermediate serving a specialised API to the app frontend via GraphQL.
+
+See the README.md of each respective project for further details, including
+installation, usage, and testing instructions.
+
+## Development
+
+Notes and guidelines for committers:
+
+- [Use tpope commit norms](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html): Capitalize commit messages, use the imperative present tense, wrap commit messages at 50 characters.
+- Check [Linear](https://linear.app/) for project backlogs, issues, discussion, etc.
```

Comment:
Are Linear backlogs public? If not it's a bit miss-leading to point to it in a public repo.

Reply:
Oh, this repo is public isn't it. Yes that's true. I'll remove it.